modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/running-gitops-control-plane-workloads-on-infrastructure-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
```

Comment:
Similarly to this module introduction: "You can move the {gitops-shortname} control plane workloads installed by the {gitops-title} to the infrastructure nodes." , the explanation why is listed in the introduction of the assembly and is not included in the module.  Normally, I would include the similar statement in each module as well, but here the user would have to be familiar with benefits already, that is, the assembly introduction here is essential, otherwise the user would not consider moving it to the infra nodes at all. I hope that explains why I did not repeat that same information here. Thank you!